cmd/mungedocs/mungedocs.go

```diff
@@ -199,4 +199,8 @@ func main() {
 		fmt.Fprintf(os.Stderr, "FAIL: changes needed but not made due to --verify\n")
 		os.Exit(1)
 	}
+	if changesNeeded {
```

Comment:
join this with teh above block:

if changesNeeded {
  if *verfiy {
     msg 1
  else
    msg 2
  }
  exit(1)
}